python/lsst/pipe/tasks/postprocess.py

```diff
@@ -1486,3 +1485,187 @@ def run(self, visitSummaries):
 
         outputCatalog = pd.DataFrame(data=visitEntries)
         return pipeBase.Struct(outputCatalog=outputCatalog)
+
+
+class WriteForcedSourceTableConnections(pipeBase.PipelineTaskConnections,
+                                        dimensions=("instrument", "visit", "detector", "skymap", "tract")):
+
+    inputCatalog = connectionTypes.Input(
+        doc="Primary multi-epoch, per-detector, forced photometry catalog. "
```

Comment:
Any time I say forced photometry I try to add prepend it with the modifier "multi-epoch" or "multi-band" since we do both and it's confusing. Let's call it single-epoch here.